install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -127,8 +127,10 @@ spec:
           - --lightstepAccessToken
           - {{ $.Values.global.tracer.lightstep.accessToken }}
           - --lightstepSecure={{ $.Values.global.tracer.lightstep.secure }}
+          {{- if $.Values.global.tracer.lightstep.secure }}
```

Comment:
```suggestion
          {{- if $.Values.global.tracer.lightstep.cacertPath }}
```

Reply:
The [documentation](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/values.yaml#L339) says that "if secure is true, this [cacertPath] is required." My understanding here was that only if `secure` is true is the `cacertPath` actually used / needed. This means that if someone sets `secure` to true but doesn't set `cacertPath` the validation error is good. If someone sets `secure` to false but does set `cacertPath` it won't be used and thus can still be excluded.

Reply:
makes sense to me